fml/mapping.cc

```diff
@@ -64,6 +64,14 @@ std::unique_ptr<FileMapping> FileMapping::CreateReadExecute(
   return mapping;
 }
 
+MappingReleaseProc FileMapping::GetReleaseProc() {
+  MappingReleaseProc proc = [](const void* ptr, void* context) {
+    auto* mapping = static_cast<fml::FileMapping*>(context);
+    mapping->~FileMapping();
+  };
+  return proc;
+}
+
```

Comment:
I believe the idiom for this is ot use a `NonOwnedMapping`.

Directly calling a destructor is pretty weird and should be avoided.

Reply:
In particular, NonOwnedMapping can be handed a release proc to run, and for this case we'd give it the SkData release proc.

Reply:
I think that is backwards of what we need though - non owned mapping calls the release proc given in its destructor but the problem is that with the release required to pass the mapping to skdata, the destructor won't get called - right?